src/exposure.jl

```diff
@@ -0,0 +1,161 @@
+const DEGRADATION_RATE = Param(0.5; bounds=(0.0, 100.0))
+const PULSE_LEVEL = Param(20.0; bounds=(0.0, 100.0))
+const POPULATION_THRESHOLD = Param(20.0; bounds=(0.0, 100.0))
+const CROP_OCCUPANCY = Param(5.0; bounds=(0.0, 10.0))
+const ROTATION_NUMBER = Param(1.0; bounds=(0.0, 1000.0))
+const ROTATION_SIZE = Param(1.0; bounds=(0.0, 1000.0))
+
+"""
+Extends CellRule for rules of exposure dynamics
+
+For best performance these should be chained with other
+CellRule or following an NeighborhoodRule.
+"""
+
+abstract type Exposure{R,W} <: CellRule{R,W} end
+
+"""
```

Comment:
Add a signature for the constructor in the doc string, something like:

```julia
Threshold_Exposure{R,W}(;
    crop,  pulseLevel,  popThreshold,  degradationRate,  timestep,
)
```
See the other files for examples.